src/sections/home/Home.Hero.tsx

```diff
@@ -1,59 +1,188 @@
-import React from "react";
-import styled from "@emotion/styled";
+import React from 'react';
+import styled from '@emotion/styled';
+import Lottie from 'react-lottie';
 
-import Section from "@components/Section";
-import Heading from '@components/Headings'
-import Transitions from '@components/Transitions'
-import Button from '@components/Button'
+import Section from '@components/Section';
+import Transitions from '@components/Transitions';
+import ScrollIndicator from '@components/ScrollIndicator';
+import ArrowButton from '@components/Button/Button.Arrow';
 
-import media from "@styles/media";
+import media from '@styles/media';
+import { IColorThemeProps, IColorTheme } from '@types';
+import mediaqueries from '@styles/media';
+
+import animationData from '../../assets/17059-galacticons-animated.json';
+import { useThemeUI, useColorMode } from 'theme-ui';
 
 const HomeHero: React.FC = () => {
+  const themeContext = useThemeUI();
+  const [colorMode] = useColorMode();
+  const isDark = colorMode === `dark`;
+  const theme: IColorTheme = themeContext.theme as any;
+  const lottieOptions = {
+    loop: true,
+    autoplay: true, 
+    animationData: animationData,
+    rendererSettings: {
+      preserveAspectRatio: 'xMidYMid slice'
+    }
+  };
   return (
     <Section>
-      <ContentContainer>
-        <Transitions.CSS.FadeIn>
-          <Heading.h1>
-            UOSC - The club to build open source projects, compete
-            against each other, and design great products.
-          </Heading.h1>
-          <MainText>
-            We are a team of dedicated students on a mission to help
-            raise and amplify talent at uOttawa.
-          </MainText>
-          <Button
-            text="Coming soon..."
-            isDisabled={true}
+      <HeroContainer>
+        <ContentContainer>
+          <Transitions.CSS.FadeIn>
+            <MainText theme={theme}>
+              UOSC, empowering open-source projects and competitive programming
```

Comment:
Thoughts on: "Empowering any student with a passion for software to achieve more."